src/main/java/be/seeseemelk/mockbukkit/entity/SlimeMock.java

```diff
@@ -0,0 +1,65 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Slime;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.UUID;
+
+public class SlimeMock extends MobMock implements Slime
+{
+
+	private int size = 1;
+	private boolean canWander = true;
+
+	/**
+	 * Constructs a new {@link SlimeMock} on the provided {@link ServerMock} with a specified {@link UUID}.
+	 *
+	 * @param server The server to create the entity on.
+	 * @param uuid   The UUID of the entity.
+	 */
+	public SlimeMock(@NotNull ServerMock server, @NotNull UUID uuid)
+	{
+		super(server, uuid);
+	}
+
+	@Override
+	public int getSize()
+	{
+		return this.size;
+	}
+
+	@Override
+	public void setSize(int sz)
```

Comment:
```suggestion
	public void setSize(int size)
```
Please use descriptive Parameter names and try not to abbreviate

Reply:
Commit 46013cb